### PR TITLE
TASK: Add options to text input and area

### PR DESCRIPTION
**Pull request by samuelhauser**

### CODE REVIEW

**Comment by samuelhauser** [comment]
checks failed because of #784

File: packages/neos-ui-editors/src/Boolean/index.js [hunk 1]
```diff
@@ -47,7 +46,8 @@ BooleanEditor.propTypes = {
     label: PropTypes.string.isRequired,
     value: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
     commit: PropTypes.func.isRequired,
-    isDisabled: PropTypes.bool
+    isDisabled: PropTypes.bool,
```
Comment (mstruebing) [inline packages/neos-ui-editors/src/Boolean/index.js]: isn't this already in the options?

Comment (mstruebing) [inline packages/neos-ui-editors/src/Boolean/index.js]: This line can be deleted I think.

Comment (mstruebing) [inline packages/neos-ui-editors/src/Boolean/index.js]: This line can be removed as the disabled state comes from the options and isDisabled is also never used.

**Pull request closed without merging**